edenai_apis/utils/file_handling.py

```
@@ -0,0 +1,60 @@
+
+        file_wrapper_params = {
+            "file_url": file_url,
+            "file_info": FileInfo(
+                file_size=file_size,
+                file_mimetype=file_type,
+                file_extension=file_type.split("/")[-1],
+            ),
+        }
+
+        if file_size == 0:
+            raise Exception("File size is 0")
```

Ruff (0.14.3)

Create your own exception

(TRY002)